Tests/Server.py

```diff
@@ -210,6 +210,18 @@ def test_get_transaction_log_entries_from_today(self):
         today = datetime.datetime.combine(datetime.date.today(), datetime.time(0, 0))
         entries = self.tm1.server.get_transaction_log_entries(reverse=True, since=today)
         self.assertTrue(len(entries) > 0)
+        for entry in entries:
+            entry_timestamp = dateutil.parser.parse(entry['TimeStamp'])
+            # all the entries should have today's date
+            entry_date = entry_timestamp.date()
+            today_date = datetime.date.today()
+            self.assertTrue(entry_date == today_date)           
+    
+    def test_get_message_log_entries_from_today(self):
+        # get datetime from today at 00:00:00
+        today = datetime.datetime.combine(datetime.date.today(), datetime.time(0, 0))
+        entries = self.tm1.server.get_message_log_entries(reverse=True, since=today)
```

Comment:
Here you are passing the argument as `since` but in the function its called `start_time` and `end_time`.

In the `get_transaction_log_entries` filtering method we call the argument `since`.  I think we should be consistent. Either we call them all `since` and `until` or `start_time` and `end_time`.